recipes/numactl-cos6-x86_64/meta.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+  name: numactl-cos6-x86_64
+  version: 2.0.9
+
+source:
+  - url: http://mirror.centos.org/centos/6.10/os/x86_64/Packages/numactl-2.0.9-2.el6.x86_64.rpm
+    sha256: 70e58c3b01ce62d90ef0e3d9f1c0d1d3efaa4d237b3b821ee7d1e791553d82a7
+    folder: binary
+  - url: http://vault.centos.org/6.10/os/Source/SPackages/numactl-2.0.9-2.el6.src.rpm
+    folder: source
```

Comment:
Currently there is no good way to determine the checksum of the source RPM. Please see issue ( https://github.com/conda/conda-build/issues/3552 ) for details.

Reply:
I've gone ahead and dropped the source RPM as it is not used.